lib/util/Util.ts

```diff
@@ -72,7 +72,7 @@ export default class Util {
                             type:     component.type
                         };
                     }
-                } else if (component.type === ComponentTypes.SELECT_MENU) {
+                } else if (component.type === ComponentTypes.STRING_SELECT) {
```

Comment:
This needs to both be compressed, and changed to support the new component types